### ISSUE TITLE
Template string parsing error

### ISSUE THREAD

**Issue by koczkatamas** [SonarSource/SonarJS#922]
**I want to report a parser bug.**

**SonarJS version:** `4.0 (build 5862)`
**SonarQube version:** `3.0.3.778`
**SonarLint version:** I think I don't use SonarLint (I am running SonarScanner directly)

**Rule key:** parsing error (`Unable to parse file:`)

**Reproducer**

Minimal test case:
```js
prop(`//`);
$(`//`);
```

Original code:
```js
$('#link1').prop('href', `https://example.com/${path}`);
$('#link2').prop('href', `https://example.com/${path2}`);
```

**Logs**

Minimal:
```
ERROR: Unable to parse file: file:///C:/Programs/sonarqube/sonartest/test.js
ERROR: Parse error at line 2 column 9:

1: prop(`//`);
2: $(`//`);
           ^
```

Original:
```
ERROR: Unable to parse file: file:///C:/Programs/sonarqube/sonartest/test.js
ERROR: Parse error at line 2 column 27:

1: $('#link1').prop('href', `https://example.com/${path}`);
2: $('#link2').prop('href', `https://example.com/${path2}`);
                             ^
3:
```

**Expected behavior**

File is parsed successfully, no parsing error is thrown.

**Notes**

I think it has something to do with comments (`//`) inside template strings (strings with backticks) and it also requires `$` to be used later (maybe because the dollar sign can be part of a template string, eg. `${path}`).

If I submitted the issue under a wrong issue tracker, then please advise which is the correct place (eg. if you use a third-party JS parser library). Thanks!

**Comment by vilchik-elena** [comment]
Thanks for reporting! Looks like PR (#921) I've just opened fixes your problem. 
Could you confirm that trying this build?
[sonar-javascript-plugin-4.1.0.6010.jar.zip](https://github.com/SonarSource/SonarJS/files/1699119/sonar-javascript-plugin-4.1.0.6010.jar.zip)

**Comment by koczkatamas** [comment]
Yes, I can confirm that the issue is fixed in version 4.1.0.6010.

Thanks!
